product_docs/docs/biganimal/release/using_cluster/terraform_provider.mdx

```diff
@@ -6,37 +6,36 @@ BigAnimal’s [Terraform provider](https://github.com/EnterpriseDB/terraform-pro
 
 The current version of the Terraform provider offers modules for creating, reading, updating, and deleting clusters and regions.
 
+The Terraform provider is licensed under the [MPL v2](https://www.mozilla.org/en-US/MPL/2.0/).
+
 !!!note
-   We provide support for the BigAnimal Terraform provider itself and not for the underlying environment. 
+   EnterpriseDB provides support for the BigAnimal Terraform provider itself and not for the underlying environment. 
 
 ## Prerequisites
 To use Terraform with BigAnimal, you need: 
 
 - A BigAnimal account with an organization set up. If you don't already have a BigAnimal account, see [Getting started with the BigAnimal free trial](/biganimal/latest/).
 
-- [Terraform](https://www.terraform.io/downloads) downloaded and installed.
-- An API token for use within the Terraform application. See [Getting an API Token](getting-an-api-token).
+- [Terraform](https://www.terraform.io/downloads) (version 0.13*x* or later) downloaded and installed.
+- A BigAnimal API token for use within the Terraform application. See [Getting an API Token](getting-an-api-token).
 
-## Configure the provider
+## Example usage
 
-```
-terraform {
-  required_providers {
-    biganimal = {
-      source = "enterprisedb/biganimal"
-      version = "0.3.1"
-      ba_token "<redacted>"
-    }
-  }
+```terraform
+# Configure the BigAnimal Provider
+provider "biganimal" {
+  ba_token = "<redacted>"
+  //ba_api_uri   = "https://portal.biganimal.com/api/v2" // Optional
 }
+# Manage the resources
 ```
 
-See [Provider Configuration](https://developer.hashicorp.com/terraform/language/providers/configuration).
```

Comment:
Why are you taking this out? I think it would be helpful to users to keep this in.